frame.go

```diff
@@ -87,6 +89,35 @@ func (f *Frame) DataBytes() ([]byte, error) {
 
 }
 
+// SetDataBytes fills the frame data with given raw data. The data must match the pixel format and the dimensions of the frame.
+func (f *Frame) SetDataBytes(data []byte) (err error) {
+
```

Comment:
Could you add the `if f.Height() > 0 && f.Width() > 0` logic here as well ?

In fact, could you add a `Frame.isImage() bool` unexported function that returns exactly that, and use it in both `DataBytes` and `SetDataBytes`?